components/Editor.js

```diff
@@ -93,6 +93,16 @@ class Editor extends React.Component {
   updateCode = code => this.updateState({ code })
   updateWidth = width => this.setState({ widthAdjustment: false, width })
 
+  calculateExportImageBaseWidth = offsetWidth => {
+    if (this.state.widthAdjustment) return offsetWidth
+
+    const settingWidth = parseInt(this.state.width)
+    const isOverWidth = offsetWidth < settingWidth
+    if (!isOverWidth) return offsetWidth
+
+    return settingWidth + parseInt(this.state.paddingHorizontal)
```

Comment:
When the set width is larger than the displayed width, the width of the exported image is calculated using the set width.